packages/website/blog/2023-07-09-announcing-typescript-eslint-v6.md

```diff
@@ -628,19 +628,23 @@ import * as TSESLint from '@typescript-eslint/utils/dist/ts-eslint';
 
 That presents a few issues for developers:
 
-- It can be unclear which of many potential import paths to use
-- TypeScript sometimes suggests importing types or values meant to be private
-- Consumers using deep import paths can be broken by internal refactors that rename files
+- It can be unclear which of many potential import paths to use.
+- TypeScript sometimes suggests importing types or values meant to be private.
+- Consumers using deep import paths might get broken code after internal refactors rename files.
```

Comment:
Speaking directly to this one: I still think the original phrasing is more clear:

* The "can" and "might" mean about the same
* It's not the code that's broken, it's the runtime and import paths

Reply:
ok, i reverted the change